features/steps/start_request.py

```diff
@@ -1,22 +1,44 @@
+from intake.models import *
+from intake.services import *
```

Comment:
Could we specify the imports instead of using `*`? I think that's somewhat discouraged to avoid overwriting something accidentally.